### PR TITLE
Remove unused imports

### PR DESCRIPTION
**Pull request by davotoula**
Cleaner code. First dabble in Shufflepuff.

### CODE REVIEW

**Comment by DanielKrawisz** [comment]
Thank you!

File: shuffler/src/main/java/com/shuffle/player/proto/Proto.java [hunk 1]
```diff
@@ -153,7 +153,7 @@ public static Phase valueOf(
 
     private final int value;
 
-    private Phase(int value) {
+    Phase(int value) {
```
Comment (DanielKrawisz) [inline shuffler/src/main/java/com/shuffle/player/proto/Proto.java]: Please revert this change. Proto.java is an autogenerated file, so you shouldn't edit it.

File: shuffler/src/main/java/com/shuffle/player/proto/Proto.java [hunk 2]
```diff
@@ -314,7 +314,7 @@ public static Reason valueOf(
 
     private final int value;
 
-    private Reason(int value) {
+    Reason(int value) {
```
Comment (DanielKrawisz) [inline shuffler/src/main/java/com/shuffle/player/proto/Proto.java]: This too.

**Comment by davotoula** [comment]
fixed
